### ISSUE TITLE
Lootbags possibly not differentiating fake and real player kills

### ISSUE THREAD

**Issue by minakitty** [Malorolam/LootBags#156]
This is going to be a cross posted bug, because I can't tell if it's Lootbags or Woot.  Lootbag is supposed to be configured to only drop when mobs are killed by an actual player, but when I made my first Woot grinder, it still generated loot tables as though a real player were doing the kills, and I've confirmed that it's actually outputting as such to the output chest.  It seems possible that Lootbag is not discriminating properly, although it's also possible that Woot is not identifying the kill method properly (hence the crosspost)

I'm attaching the current copy of lootbags.cfg
[lootbags.txt](https://github.com/Malorolam/LootBags/files/2445080/lootbags.txt)

**Comment by minakitty** [comment]
https://github.com/Ipsis/Woot/issues/387 - Related Woot bug

**Comment by minakitty** [comment]
So I just closed the Woot ticket after learning that I could blacklist  lootbags in Woot.  So a non-ideal solution, as it only fixes that one grinder option  Ipsis took a peek at your code and suspected that you may only be checking for EntityPlayer, which FakePlayer is a part of, but they weren't sure.  Or it could be something else entirely

**Comment by Malorolam** [comment]
Yup, looks like the problem is on my end here, one of those EntityPlayers should indicate either not a FakePlayer or the real player.  Looks like somewhere the class I was using for it got removed and it changed to EntityPlayer.  Been meaning to rework that section for a while since it's so messy.  Should be fixed in the next version.
